src/components/Post/post.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import Inputs from '../../components/Inputs/inputs';
+
+import axios from 'axios';
+
+class Post extends Component {
+
+  constructor(){
+    super();
+    this.state = {
```

Comment:
These are not needed anymore, we could just declare the `state` directly like 

`state = { name: "" }`

Reference: [The constructor is dead, long live the constructor!](https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599)

Reply:
good to know, thnx